Create ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md (#2103)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md

---
date: ""

> title: add info on how to style shell application via plugin (#6177)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add information on how to style the shell application using a plugin
```

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58246
version: 1020.0.0
---

> add info on how to style shell application via plugin (#6177)

Author: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility in customizing the look and feel of the shell application, a new plugin mechanism has been introduced. This change allows developers to create plugins that can modify the styling of various elements in the shell application. The impact of this change is that developers now have more control over the visual appearance of the shell application and can tailor it to their specific needs or branding requirements without having to modify the core application code.
```

> Documentation has been added on how to style the shell application via plugins. 

Reviewer: @DawidSAG Can you please providedetails where it has been added?

Reviewer 2: @BeateRixen can I just add link to the codex? https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin
```
Documentation has been added on how to style the shell application via plugins. 
For details, refer to [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).
```

Reviewer: Yes, exactly :-)

---
date: ""
title: Documented shell application styling via plugins
product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58246
version: 1020.0.0
---
Documentation has been added on how to style the shell application via plugins. 

> For details, refer to [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).

Reviewer: ```suggestion
For details, refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).
```
